locale/en/blog/announcements/nodejs-foundation-momentum-release.md

```diff
@@ -0,0 +1,42 @@
+---
+title: The Node.js Platform and Node.js Foundation Continue to Grow
+date: 2016-11-30T12:00:00.000Z
+category: Annoucements
+slug: node.js-foundation-momentum-release
+layout: blog-post.hbs
+---
+
+> With 5.7 million users; increased community participation and a solid Foundation backing: 2016 was a good year for the platform.
+
+**NODE.JS INTERACTIVE 2016, AUSTIN, TX., Nov. 30, 2016** — [The Node.js Foundation](https://nodejs.org/en/foundation/), a community-led and industry-backed consortium to advance the development of the Node.js platform, today announced [Snyk](https://snyk.io) as a Silver Member, major community and code growth, and the opening of an expansive Node.js user survey.
```

Comment:
While I like to have 80 chars per line, most of the markdown files in this repo don't follow this practice.